### PR TITLE
emacs: open file at line number

### PR DESCRIPTION
**Pull request by ryanolsonx**
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/common/emacs.md [hunk 1]
```diff
@@ -14,3 +14,7 @@
 - Exit emacs (save buffers and terminate):
 
 `Ctrl + X, Ctrl + C`
+
+- Open a file in emacs at a line number:
```
Comment (zdroid) [inline pages/common/emacs.md]: Maybe `at a specified line number` would read better?

Comment (ryanolsonx) [reply pages/common/emacs.md]: I like that.  I'll make the change.

File: pages/common/emacs.md [hunk 2]
```diff
@@ -14,3 +14,7 @@
 - Exit emacs (save buffers and terminate):
 
 `Ctrl + X, Ctrl + C`
+
+- Open a file in emacs at a specified line number:
+
+`emacs +{{line number}} {{filename}}`
```
Comment (zdroid) [inline pages/common/emacs.md]:
```suggestion
emacs +{{line_number}} {{path/to/file}}
```

It'd be good if you could change the other `{{filename}}` to `{{path/to/file}}` as well.

Comment (ryanolsonx) [reply pages/common/emacs.md]: For sure!

**Comment by ryanolsonx** [comment]
The last push after approval was to change "line number" to "line_number"

**Comment by zdroid** [comment]
You don't have to force-push, by the way. Squash and merge is performed on the commits, so only one commit will be merged.

**Comment by ryanolsonx** [comment]
> You don't have to force-push, by the way.

Good to know.  I'm used to a work environment where we simply merge a branch.  So I try to keep my commits tidy.

**Pull request merged**